src/Microsoft.AspNetCore.All/Microsoft.AspNetCore.All.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <TargetFramework>netstandard1.6</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
I *think* this is fine, but I'd like @DamianEdwards to confirm this is ok. This means the metapackage can't be used on .NET Framework, but I think for 2.0 that's fine.